.github/workflows/docker.yml

```
@@ -0,0 +1,84 @@
+on:
+  schedule:
+    - cron: '0 7 */7 * *' # This should run once a week
+
+name: Docker images
+
+jobs:
+  build-images:
+    name: Build & push Docker images
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```
the -latest image got deprecated

---

AFAIK, it is not, it will use Ubuntu 20.04 soon instead of Ubuntu 18.04 https://github.com/actions/virtual-environments/issues/1816

---

It shouldn't be used anymore. Because -latest doesn't make any sense, latest what exactly? hence why github decided to stop serving that image someday.

---

> latest what exactly?

Latest Ubuntu version, currenty is a shorthand for `ubuntu-18.04`, but as said in that issue, soon will be a shorthand for `ubuntu-20.04`. I think on it like the `latest` tag on Docker

Anyways, I'll change to `ubuntu-20.04` if you want it

---

No really, the -latest tag will be removed. You can trust me on that :)